core/data/src/main/java/com/dev/innerview/core/data/di/DataModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.dev.innerview.core.data.di
+
+import com.dev.innerview.core.data.repository.InnerViewRepositoryImpl
+import com.dev.innerview.core.data_api.InnerViewRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class DataModule {
+
+    @Binds
+    abstract fun bindsSettingsRepository(
+        repository: InnerViewRepositoryImpl,
```

Comment:
🚫 **[ktlint]** <standard:function-signature> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>No whitespace expected between last parameter and closing parenthesis
<!-- __reviewdog__:ChBlMzQyYTlhMzdiZjYzNzllEgZrdGxpbnQ= -->